db/system_keyspace.cc

```diff
@@ -287,6 +287,24 @@ schema_ptr raft_config() {
     return schema;
 }
 
+schema_ptr repair_history() {
+    static thread_local auto schema = [] {
+        auto id = generate_legacy_id(NAME, REPAIR_HISTORY);
+        return schema_builder(NAME, REPAIR_HISTORY, std::optional(id))
+            .with_column("table_uuid", uuid_type, column_kind::partition_key)
+            .with_column("repair_time", timestamp_type, column_kind::clustering_key)
+            .with_column("repair_uuid", uuid_type, column_kind::clustering_key)
+            .with_column("range_start", long_type, column_kind::clustering_key)
+            .with_column("range_end", long_type, column_kind::clustering_key)
```

Comment:
Add comments to indicate whether it's inclusive or exclusive.

Reply:
ok.

Reply:
done